.cypress/integration/7_app_analytics.spec.js

```diff
@@ -85,15 +87,15 @@ describe('Creating application', () => {
     cy.get('[data-test-subj="createButton"]').should('not.be.disabled');
     cy.get('[data-test-subj="createAndSetButton"]').should('be.disabled');
     expectMessageOnHover('createAndSetButton', 'Log source is required to set availability.');
-    cy.get('[data-test-subj="searchAutocompleteTextArea"]').focus().type(baseQuery, {delay: TYPING_DELAY});
+    cy.get('[data-test-subj="searchAutocompleteTextArea"]').focus().type('      ' + baseQuery);
```

Comment:
<img width="1172" alt="Screenshot 2023-06-29 at 2 23 47 PM" src="https://github.com/opensearch-project/dashboards-observability/assets/105462877/afd2141f-5c6f-4289-873d-c1690a89783a">
This field seemed to incorrectly type regardless of delay applied. The error always occurred in the first 3-5 letters, adding whitespace allows it to execute correctly and the whitespace is ignored in the application. This greatly increases run-speed as well  as preventing the mis-typed log source.

Reply:
Good to know this hack 😄 Also, did you try looking deeper on why it fails to type correctly?

Reply:
I wasn't able to find any specific cause for the mis-typing just that it is a known issue and most people add very long delays to attempt to fix it. This field specifically was only encountering the error at the start after testing a lot of possible fixes / ideas I had this seemed to be the only 100% reliable one.